python/torch_mlir_e2e_test/test_suite/slice_like.py

```diff
@@ -481,3 +481,47 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: NarrowVerticalTest2())
 def NarrowVerticalTest2_basic(module, tu: TestUtils):
     module.forward(tu.rand(6,4))
+
+# ==============================================================================
+
+class SliceCopy_Module(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([10, 4, 4], torch.float32, True),
```

Comment:
do these not work with dynamic dims?

Reply:
The negative test does, but the positive one fails because the `Arange` op has a static size while the others have dynamic sizes. Do you think it's necessary to have a dynamic test when the sizes are static at this point in the pipeline?

Reply:
If the failure is due to lack of support for things in a different op (`Arange` in this case), then it's fine to leave as static. However, if tests do pass dynamically, that should always be preferred. I would change the negative dim one to dynamic.

Can you also file a bug for the error you're seeing with `Arange`, so that we can remember that it's a known issue? Things should not stop working if one argument has static information.